tests/entities/bootconfig/categories/AppearancePartTest.java

```diff
@@ -0,0 +1,82 @@
+package entities.bootconfig.categories;
+
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AppearancePartTest {
+        // Test om te controleren of de get_price methode de juiste String retourneert
+        @Test
+        void test_getPrice_returnsCorrectString() {
```

Comment:
```suggestion
        void test_get_price_returns_correct_str() {
```
Fix naming convention, we use snake_case